static/src/js/builder/index.jsx

```diff
@@ -21,7 +21,8 @@ const ancestorCodes = treeTables
   .flat();
 const visiblePaths = hierarchy.initiallyVisiblePaths(
   ancestorCodes,
-  codeToStatus
+  codeToStatus,
+  1
```

Comment:
Is there a way to make this a kwarg/named argument in JS?  (without going the full on Object route at least)

Reply:
Not according to https://stackoverflow.com/questions/11796093/is-there-a-way-to-provide-named-parameters-in-a-function-call-in-javascript.

Agree, it's unsightly.